### PR TITLE
test/unittest_bluefs: check whether rmdir success

### PR DESCRIPTION
**Pull request by ShiqiCooperation**
Signed-off-by: shiqi <1454927420@qq.com>

### CODE REVIEW

**Comment by ShiqiCooperation** [comment]
I see plain text. 
Community test case shows: no ceph.conf file. I think there is a problem with the community test case environment

**Comment by ShiqiCooperation** [comment]
@liewegas

**Comment by varadakari** [comment]
Not the problem with ceph.conf. Here is the error.
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/objectstore/test_bluefs.cc:391: Failure
      Expected: 0
To be equal to: fs.rmdir(dir)
      Which is: -39
[  FAILED  ] BlueFS.test_simple_compaction_sync (1455 ms)

Would you please debug, what went wrong? And it if passes in your testing?

**Comment by ShiqiCooperation** [comment]
ok, all test passed!

File: src/test/objectstore/test_bluefs.cc [hunk 1]
```diff
@@ -435,13 +435,13 @@ TEST(BlueFS, test_simple_compaction_async) {
     for (int i=0; i<10; i+=2) {
        string dir = "dir.";
        dir.append(to_string(i));
-       for (int j=0; j<10; j+=2) {
+       for (int j=0; j<10; j++) {
```
Comment (varadakari) [inline src/test/objectstore/test_bluefs.cc]: Don't remember the reason behind not removing all the files, may be want to fail the rmdir? If we are adding an assert to remove all, please remove the comment above.

File: src/test/objectstore/test_bluefs.cc [hunk 2]
```diff
@@ -382,13 +382,13 @@ TEST(BlueFS, test_simple_compaction_sync) {
...
```
Comment (varadakari) [inline src/test/objectstore/test_bluefs.cc]: Please remove the comment above, if we are removing all the files.

**Comment by liewegas** [comment]
retest this please

**Comment by xiexingguo** [comment]
http://pulpito.ceph.com/xxg-2017-09-01_07:17:41-rados-wip-xxg-testing-2017-09-01-1357-distro-basic-smithi/
http://pulpito.ceph.com/xxg-2017-09-03_12:51:11-rados-wip-xxg-testing-2017-09-01-1357-distro-basic-smithi/

retest this please

**Comment by xiexingguo** [comment]
retest this please

**Pull request merged**